Library/test-helpers/lib.sh

```diff
@@ -1253,6 +1253,92 @@ limeCreateTestDir() {
 }
 
 
+true <<'=cut'
+=pod
+
+=head2 limeCopyKeylimeFile
+
+Copy file to actual dir from specified path or download file
+from repository to actual dir.
+
+    limeCopyKeylimeFile
+
+=over
+
+=back
+
+Returns 0 when the when copy or download of file to actual dir is succesfull.
+
+=cut
+
+limeCopyKeylimeFile(){
+
+    local OPTION=$1
+    local NAME=$2
+
+    if [ ${OPTION} == "-s" ] ||[ ${OPTION} == "--source" ]; then
+        # get filepath to the variblae and if file exist, copy file to actual dir
+        local FILEPATH=$(limeGetKeylimeFilePath -s ${NAME})
+        if grep -q ${NAME} <<< ${FILEPATH} ; then
+            cp ${FILEPATH} .
+        else
+            echo "File is not in installed files!"
+            return 1
+        fi
+    elif [ ${OPTION} == "-i" ] ||[ ${OPTION} == "--installed" ]; then
+        # get filepath to the variblae and if file exist, copy file to actual dir
+        local FILEPATH=$(limeGetKeylimeFilePath -i ${NAME})
+        if grep -q ${NAME} <<< ${FILEPATH} ; then
+            cp ${FILEPATH} .
+        else
+            echo "File is not in installed files!"
+            return 1
+        fi
+    elif [ ${OPTION} == "-d" ] || [ ${OPTION} == "--download" ]; then
```

Comment:
I am not sure we should have `-d` option at all. Probably the most appropriate (matching to the former code in `measured-boot-policy-sanity/test.sh`) usage would be:
 - if found in sources, copy file from sources
 - otherwise download from upstream code
 So in the end we would have only `-i` and `-s`.